examples/fastify/__integration-tests__/fastify.spec.ts

```diff
@@ -308,16 +298,16 @@ describe('fastify example integration', () => {
 
       await slowFieldResolverInvoked.promise;
       abortController.abort();
-      await expect(response$).rejects.toMatchInlineSnapshot(`DOMException {}`);
+      await expect(response$).rejects.toMatchInlineSnapshot(
+        `[AbortError: The operation was aborted]`,
+      );
```

Comment:
@ardatan It seems like Node.js global fetch is failing the resource leak tests on Node.js 18. So I tried importing `fetch` from '@whatwg-node/fetch', however, it is rejecting with AbortError instead of DOMExecption. 🤔 

<img width="448" alt="image" src="https://github.com/dotansimha/graphql-yoga/assets/14338007/f865dcfd-f4bf-4683-92d3-29ee3e6751e4">

DOMException already exists on Node.js 18, so it should be safe to use that instead?

Reply:
I will merge this anyways, as it is not related really to these tests.